praw/models/reddit/subreddit.py

```diff
@@ -404,6 +405,31 @@ def quaran(self):
         """
         return SubredditQuarantine(self)
 
+    @cachedproperty
+    def rules(self):
+        """Provide an instance of :class:`.SubredditRules`.
+
+        Use this attribute for interacting with a subreddit's removal reasons.
+        For example, to list all the removal reasons for a subreddit:
+
+        .. code-block:: python
+
+            for rule in reddit.subreddit('AskReddit').rules:
+                print(rule)
```

Comment:
I'm thinking it might be better to have the way these are accessed be 

```python
for rule in reddit.subreddit('AskReddit').rules():
    print(rule)
```

(with `()` for a call expression).

I think this because it's more like the previous behavior. I'm not certain about this though, so I'd like you hear your thoughts.

Reply:
I think we could add a deprecation to this method. Calling it just returns the iter method.